### PR TITLE
fix: detect container when running via podman

### LINKED ISSUE #2602: Running via podman is not recognized as "inside container"
If I want to run [the official Docker image](https://hub.docker.com/r/mozillaiot/gateway) via podman instead of Docker, the app doesn't recognize, that it's running inside a container.

Basically [`is_container() in run-app.sh`](https://github.com/WebThingsIO/gateway/blob/d84c0561385876269d703445f3944911c3870624/run-app.sh#L9-L23) returns `false` instead of `true`.

### PR DESCRIPTION
**Pull request by foodev**
According to https://docs.podman.io/en/latest/markdown/podman-run.1.html
podman creates the file `/run/.containerenv` to indicate to programs
they are running in a container.

resolves #2602

### CODE REVIEW

File: run-app.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ args=""
 start_task="run-only"
 
 is_container() {
-  if [ -f /.dockerenv ]; then
+  if [ -f /.dockerenv ] || [ -f /run/.containerenv ]; then
```
Comment (mrstegeman) [inline run-app.sh:10]: Please break this into a separate block, just to keep things consistent.

**Review by mrstegeman — CHANGES_REQUESTED** [review]
There’s also a check in the source that needs to be updated: https://github.com/WebThingsIO/gateway/blob/4606c6d73854afae90cc92a8b07ae1c18670763e/src/platform.js#L89

**Comment by foodev** [comment]
https://github.com/WebThingsIO/gateway/blob/4606c6d73854afae90cc92a8b07ae1c18670763e/src/platform.js#L89

Then I would change the name to `isContainer`, too. This way it's consistent with the Bash script.

Furthermore I think the user-agent should contain `(container)` instead of `(docker)`.

https://github.com/WebThingsIO/gateway/blob/d84c0561385876269d703445f3944911c3870624/src/utils.js#L70

Is that OK?

**Comment by mrstegeman** [comment]
That sounds good to me.

File: src/platform.js [hunk 2]
```diff
@@ -84,10 +84,11 @@ function getArchitecture() {
 }
 
 /**
- * Determine whether or not we're running inside Docker.
+ * Determine whether or not we're running inside a contianer (e.g. Docker).
```
Comment (mrstegeman) [inline src/platform.js:87]: nit: `container`

**Review by mrstegeman — CHANGES_REQUESTED** [review]

**Review by mrstegeman — APPROVED** [review]

**Pull request merged**